Runes.xcodeproj/project.pbxproj

```diff
@@ -713,6 +726,7 @@
 		51DE8A2B1BAB36E600124320 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				APPLICATION_EXTENSION_API_ONLY = NO;
```

Comment:
Why revert this? We should be able to use Runes in extensions without warnings as Runes doesn't use any extension unsafe API.